src/os/tempfile.go

```diff
@@ -142,7 +142,7 @@ func joinPath(dir, name string) string {
 	return dir + string(PathSeparator) + name
 }
 
-// LastIndexByte from the strings package.
```

Comment:
oh, booger.  This typo fix broke the meaning of the comment.  Might want to amend this to give the new function name and then the old function name if you are motivated enough.